src/cmds/qdel.c

```diff
@@ -167,20 +240,33 @@ char **envp;
 		fprintf(stderr, "qdel: unable to initialize security library.\n");
 		exit(1);
 	}
-
+	
+	/* allocate enough memory to store array of server instances */
+	no_of_pbsservers = 10;
+	pbs_servers = (pbs_svr_t *)calloc(no_of_pbsservers, sizeof(pbs_svr_t));
```

Comment:
You can use add_ptr_to_array() and remove this code.

Reply:
It's not a string addition operation. we can't use that function.